uber/site_sections/emails.py

```diff
@@ -16,18 +16,28 @@ def sent(self, session, **params):
     sent.restricted = [c.PEOPLE, c.REG_AT_CON]
 
     def pending(self, session, message=''):
-        approved = {ae.subject for ae in session.query(ApprovedEmail).all()}
+        approved_subjects = {ae.subject for ae in session.query(ApprovedEmail).all()}
+
+        automated_emails = []
+        for automated_email in AutomatedEmail.instances.values():
+            automated_emails.append({
+                'doesnt_need_approval': not automated_email.needs_approval,
+                'approved': not automated_email.needs_approval or automated_email.subject in approved_subjects,
```

Comment:
Sorry, I'm not sure what you're asking exactly? which part is counting what twice?